msalautomationapp/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<network-security-config>
```

Comment:
If we could add a comment this file explaining why it is here that would be helpful. At a glance, it's unclear if this is for setting up Fiddler for for enabling these tests, or both

Reply:
Done.
<!-- Trust user added CAs while debuggable only. e.g. Fiddler -->